src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -104,10 +142,9 @@ public void putAngleOnSmartDashboard() {
 
   @Override
   public void periodic() {
+    SmartDashboard.putNumber("amp multiplier", 4 / 9);
+    SmartDashboard.putNumber("speaker multiplier", 2 / 9);
     SmartDashboard.putNumber("shield rots", m_shieldController.getEncoder().getPosition());
-    if (DriverStation.isTest()) {
```

Comment:
Why was this removed?

Reply:
not relevant (most logging intensive stuff like pid has already been tuned)